boards/arm64/renesas_rcar/gen3/common_salvator_xs_h3ulcb.dts

```diff
@@ -34,7 +34,10 @@
 	pinctrl-0 = <&sd_clk &sd_cmd &sd_ds
 			&sd_data0 &sd_data1 &sd_data2 &sd_data3
 			&sd_data4 &sd_data5 &sd_data6 &sd_data7>;
-	pinctrl-names = "default";
+	pinctrl-1 = <&sd_clk &sd_cmd &sd_ds
+			&sd_data0 &sd_data1 &sd_data2 &sd_data3
+			&sd_data4 &sd_data5 &sd_data6 &sd_data7>;
+	pinctrl-names = "default", "uhs";
```

Comment:
How this supposed to work if both pinctlr states are basically the same?

Reply:
it is the same for mmc card, because it can work only with 1.8V
but it should be different for SD cards